std/file.d

```diff
@@ -2809,7 +2809,7 @@ version(StdDdoc)
     struct DirEntry
     {
         /++
-            Constructs a DirEntry for the given file (or directory).
+            Constructs a $(D DirEntry) for the given file (or directory).
```

Comment:
I don't know if it's a bug or not that these aren't being automatically highlighted like the other symbols.